### PR TITLE
chore: bump to v0.55.0

### PR DESCRIPTION
**Pull request by JoshuaBatty**
## Description

waiting on #5871

### CODE REVIEW

**Review by kayagokalp — APPROVED** [review]

**Comment by kayagokalp** [comment]
https://github.com/FuelLabs/sway/pull/5858 -- this is marked breaking but it is breaking from 0.53.0 perspective i believe as it is continuation of #5844 . Any idea if this should be a major?

**Comment by JoshuaBatty** [comment]
True you're right, good catch. I'll change this to be a major release now.

**Review by kayagokalp — APPROVED** [review]

**Review by jjcnn — CHANGES_REQUESTED** [review]

File: docs/reference/src/documentation/introduction/installation/source.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ The `Sway toolchain` can be built directly from the [`Sway repository`](https://
 ## Installation & Updating
 
 <!-- markdown-link-check-disable -->
-In the root of the repository `/sway/<here>` build [`forc`](https://fuellabs.github.io/sway/v0.54.0/book/forc/commands/index.html) with the following command:
+In the root of the repository `/sway/<here>` build [`forc`](https://fuellabs.github.io/sway/v0.54.1/book/forc/commands/index.html) with the following command:
```
[the remark this answers is not in the document]
Comment (JoshuaBatty) [reply docs/reference/src/documentation/introduction/installation/source.md:8]: Good catch not sure how find and replace missed these, they should be 0.55.0

[the remark this answers is not in the document]
Comment (JoshuaBatty) [reply docs/reference/src/documentation/introduction/installation/source.md]: resolved in #12c9fd6

File: docs/reference/src/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # The Sway Reference
 
 <!-- markdown-link-check-disable -->
-This is the technical reference for the Sway programming language. For a prose explanation and introduction to the language, please refer to the [Sway Book](https://fuellabs.github.io/sway/v0.54.0/book/).
+This is the technical reference for the Sway programming language. For a prose explanation and introduction to the language, please refer to the [Sway Book](https://fuellabs.github.io/sway/v0.54.1/book/).
```
[the remark this answers is not in the document]
Comment (JoshuaBatty) [reply docs/reference/src/index.md]: resolved in [12c9fd6](https://github.com/FuelLabs/sway/pull/5872/commits/12c9fd6d2323bcd2d8edb7ebfb3260b22dca9844)

File: docs/reference/src/documentation/introduction/installation/index.md [hunk 3]
```diff
@@ -19,9 +19,9 @@ The supported operating systems include Linux and macOS; however, Windows is [`u
 `Cargo` may be used instead of [`Fuelup`](fuelup.md); however, the user needs to manage the toolchain themselves.
 
 <!-- markdown-link-check-disable -->
-The advantage of using `Cargo` is the installation of [`plugins`](https://fuellabs.github.io/sway/v0.54.0/book/forc/plugins/index.html) that have not been added into [`Fuelup`](fuelup.md).
+The advantage of using `Cargo` is the installation of [`plugins`](https://fuellabs.github.io/sway/v0.54.1/book/forc/plugins/index.html) that have not been added into [`Fuelup`](fuelup.md).
```
[the remark this answers is not in the document]
Comment (JoshuaBatty) [reply docs/reference/src/documentation/introduction/installation/index.md]: resolved in [12c9fd6](https://github.com/FuelLabs/sway/pull/5872/commits/12c9fd6d2323bcd2d8edb7ebfb3260b22dca9844)

**Review by jjcnn — APPROVED** [review]

**Pull request merged**